Source/LES.cpp

```diff
@@ -255,27 +322,11 @@ PeleC::getSmagorinskyLESTerm(
       }
 
       // Compute flux divergence (1/Vol).Div(F.A)
-      auto const& Lterm = LESTerm.array(mfi);
-      setV(vbox, NVAR, Lterm, 0.0);
-      {
-        BL_PROFILE("PeleC::pc_flux_div()");
-        auto const& vol = volume.array(mfi);
-        amrex::ParallelFor(
-          vbox, NVAR,
-          [=] AMREX_GPU_DEVICE(int i, int j, int k, int n) noexcept {
-            pc_flux_div(
-              i, j, k, n, AMREX_D_DECL(flx[0], flx[1], flx[2]), vol, Lterm);
-          });
-      }
+      computeFluxDiv(LESTerm, mfi, vbox, flx, volume);
 
       // Refluxing
-      if (do_reflux && reflux_factor != 0) {
-        amrex::FArrayBox dm_as_fine(
-          amrex::Box::TheUnitBox(), LESTerm.nComp(), amrex::The_Async_Arena());
-        update_flux_registers(
-          reflux_factor * dt, mfi, typ,
-          {AMREX_D_DECL(&flux_ec[0], &flux_ec[1], &flux_ec[2])}, dm_as_fine);
-      }
+      updateFluxRegistersLES(
+        reflux_factor, reflux_factor, LESTerm, dt, mfi, typ, flux_ec);
```

Comment:
It looks like the first argument here (and other calls to this function) should be `do_reflux`

Reply:
Thanks for the review. The modifications have been made.